src/hooks/useLoadGlobalLateData.js

```diff
@@ -27,10 +31,11 @@ export default function useLoadGlobalLateData() {
     const promises = [];
     const p1 = loadWalletBalanceNamesToCache();
     const p2 = dispatch(nonceManagerLoadState());
-    promises.push(p1, p2);
+    const p3 = prefetchAccountENSDomains({ accountAddress });
+    promises.push(p1, p2, p3);
```

Comment:
not related to this pr but I feel we can resolve and save ENS name of the wallet here as well